fiftyone/core/annotation/validate_label_schemas.py

```
@@ -21,6 +21,11 @@
 class ValidationErrors(ExceptionGroup):
     """Validation errors for label schemas"""
 
+    def __str__(self):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)